src/anemoi/training/diagnostics/callbacks/evaluation.py

```diff
@@ -53,33 +53,9 @@ def _eval(
         loss = torch.zeros(1, dtype=batch.dtype, device=pl_module.device, requires_grad=False)
         metrics = {}
 
-        # start rollout
-        batch = pl_module.model.pre_processors(batch, in_place=False)
-        x = batch[
-            :,
-            0 : pl_module.multi_step,
-            ...,
-            pl_module.data_indices.internal_data.input.full,
-        ]  # (bs, multi_step, latlon, nvar)
-        assert (
```

Comment:
Please leave this assert. Could be changed to  "Batch length not sufficient for requested validation rollout length!"